docker-compose.yml

```diff
@@ -195,40 +178,36 @@ services:
       CONNECT_STATUS_STORAGE_REPLICATION_FACTOR: '1'
       CONNECT_PLUGIN_PATH: ' /usr/share/java/'
 
-  akhq:
-    image: tchiotludo/akhq
-    networks:
-      - cassandra
-    ports:
-     - "18080:8080"
-    environment:
-      AKHQ_CONFIGURATION: |
-        akhq:
-          security:
-            basic-auth:
-              - username: admin
-                password: d49745a74edfd20705ccaa277209db554b6b0d78c20780a4facf87e88aee78c0
-                groups: 
-                  - admin
-          connections:
-            docker-kafka-server:
-              properties:
-                bootstrap.servers: "kafka:9092"
-              schema-registry:
-                url: "http://schema-registry:8085"
-              connect:
-                - name: "connect"
-                  url: "http://connect:8083"
+  # akhq:
```

Comment:
Remove this. Let's create a new docker-compose.debug.yaml that has all these things.